Stripe.xcodeproj/project.pbxproj

```diff
@@ -2353,23 +2353,13 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 04F39F0D1AEF2AFE005B926E /* StripeiOS Tests-Debug.xcconfig */;
 			buildSettings = {
-				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
```

Comment:
Did you delete these lines on purpose? It seems like the tests would not compile without them